pom.xml

```diff
@@ -28,12 +28,12 @@
 
```

Comment:
1. change url in issueManagement
2. change url in maven-javadoc-plugin

Reply:
Done.